api/tests/unit/environments/test_unit_environments_views_sdk_environment.py

```diff
@@ -76,7 +108,7 @@ def test_get_environment_document(
     url = reverse("api-v1:environment-document")
 
     # When
-    with django_assert_num_queries(13):
+    with django_assert_num_queries(16):
```

Comment:
I guess this increase is due to all the extra prefetch queries added to handle the identity overrides?

Reply:
Yes.